src/diff/diffCommand.ts

```diff
@@ -70,6 +70,11 @@ Exit statuses:
         "Specifies the API spec of the files being compared. Defaults to RAML. Options are RAML or OAS",
       options: ["raml", "oas"],
     }),
+    "normalize-directory-names": flags.boolean({
+      description:
+        "Normalize directory names by removing minor and patch versions. Example: 'shopper-stores-oas-1.0.16' -> 'shopper-stores-oas-1",
+      default: false,
```

Comment:
Just wondering; wouldn't the most common use case have us set this to true so that we can compare the difference between two minor versions of the same API? 

Otherwise, the diff will say the old version is removed and the new version is added and leave it at that, no? I don't think this old-version-removed/new-version-added also does a file by file compare?

Reply:
Thanks for making this change, @joeluong-sfcc !